app/src/main/java/com/endava/myendava/adapters/CalendarDayAdapter.java

```diff
@@ -0,0 +1,114 @@
+package com.endava.myendava.adapters;
+
+import android.content.Context;
+import android.graphics.Point;
+import android.util.DisplayMetrics;
+import android.view.Display;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.WindowManager;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.endava.myendava.R;
+import com.endava.myendava.models.CalendarDay;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+public class CalendarDayAdapter extends RecyclerView.Adapter<CalendarDayAdapter.CalendarViewHolder> {
+
+    public static final String TRAINING_EVENT = "Training";
+    public static final String CONFERENCE_EVENT = "Conference";
+    public static final String CERTIFICATION_EVENT = "Certification";
+
+    private final Context mContext;
+
+    private List<CalendarDay> mCalendarDayList;
```

Comment:
as discussed with Gabi also, I would like to renounce the mStuff naming and continue without the m prefix. there is an extra line between context and list

Reply:
@soundrob be aware of the naming discussion here